components/eam/src/physics/cam/zm_conv_intr.F90

```diff
@@ -10,15 +10,28 @@ module zm_conv_intr
 ! Author: D.B. Coleman
 ! January 2010 modified by J. Kay to add COSP simulator fields to physics buffer
 ! July 2015 B. Singh Added code for unified convective trasport
+! March 2023 X. Song added code for the initialization of convective microphysics
+!            and aerosol object, and the output of microphysical properties and tendencies
 !---------------------------------------------------------------------------------
    use shr_kind_mod, only: r8=>shr_kind_r8
-   use physconst,    only: cpair
+   use physconst,    only: cpair                              
```

Comment:
Looks like extra white space was added by accident?

Reply:
Extra space is removed.